src/types.rs

```diff
@@ -149,6 +155,21 @@ impl From<Network> for HWIChain {
     }
 }
 
+#[cfg(test)]
+pub const TESTNET: HWIChain = HWIChain(Network::Testnet);
+
+/// An error that will occur if rust-bitcoin adds a network that we do not know about.
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct UnknownNetworkError(Network);
```

Comment:
As far as I understand this error is defined but never used in the code. Personally, I would keep the panic in `impl IntoPy<PyObject> for HWIChain` and remove this error.